### PR TITLE
Clarify FO publication timing #735

### PR DESCRIPTION
**Pull request by fantasai**
***
<a href="https://pr-preview.s3.amazonaws.com/fantasai/w3process/pull/808.html" title="Last updated on Dec 14, 2023, 5:08 AM UTC (533cdb8)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/w3process/808/dcd212d...fantasai:533cdb8.html" title="Last updated on Dec 14, 2023, 5:08 AM UTC (533cdb8)">Diff</a>

### CODE REVIEW

**Comment by cwilso** [comment]
This seems like it would not allow the Team to attempt to resolve Formal Objections prior to making them public, which might lead to more press cycles than necessary; it also seems like the FO would be made public immediately upon the vote being filed ("upon registration"), disallowing changing the vote later.

File: index.bs [hunk 1]
```diff
@@ -2279,7 +2279,8 @@ Registering Formal Objections</h3>
 	rationales,
 	and decisions <em class="rfc2119">should</em> also be recorded.
 
-	A record of each [=Formal Objection=] against a decision regarding a publicly-available document
+	Upon registration with the [=Team=],
```
Comment (jyasskin) [inline index.bs:2282]: I guess this implies that registration needs to include the exact wording that'll become public, so the Team doesn't need to spend time going through [7.3. Changing Confidentiality Level](https://www.w3.org/2023/Process-20231103/#confidentiality-change). That's fine with me, but a delay of up to, say, a week, would also be fine with me.

**Review by jyasskin — APPROVED** [review]

**Comment by jyasskin** [comment]
@cwilso I almost made your "upon the vote being filed" comment, but then I noticed
> A [Formal Objection](https://pr-preview.s3.amazonaws.com/fantasai/w3process/pull/808.html#FormalObjection) filed during an [Advisory Committee Review](https://pr-preview.s3.amazonaws.com/fantasai/w3process/pull/808.html#advisory-committee-review) is considered registered at the close of the review period.

at the bottom of the section.

File: index.bs (hunk 1 above)
Comment (fantasai) [reply index.bs:2282]: The intention was to give some time to the Team to do that, so if this seems a bit too instantaneous, we should pick some other wording here. :)

Comment (frivoal) [reply index.bs:2282]: Based on the 2023-12-13 Process CG call:
```suggestion
	No later than when the <a href="#team-fo-mediation">relevant council is initiated<a>,
```

Comment (frivoal) [reply index.bs:2282]: (More precise guidance will be written up as a /Guide article, with an Action on @plehegar to draft that.)

**Review by jyasskin — APPROVED** [review]
I'm glad to see that the Guide will encourage the Team to publish objections somewhat faster than this requires. This amount of detail in the Process is enough for me. Thanks!

**Review by tantek — APPROVED** [review]

**Pull request merged**